src/app/src/main/resources/docker-config.properties

```diff
@@ -0,0 +1 @@
+app.api-url=http://cometr:8321
```

Comment:
I can't think of a more elegant way to do this, besides inheriting from an environment variable

Reply:
I guess we may revisit when we do the deployment scripts